app/views/course/survey/responses/_response.json.jbuilder

```diff
@@ -5,9 +5,11 @@ json.survey do
 end
 
 json.response do
-  json.(response, :id, :submitted_at)
+  json.(response, :id, :submitted_at, :creator)
```

Comment:
This will serialize the whole `User` instance and send it over, including fields like `authentication_token`. Better to extract only the fields that we need.

Also good to update the documentation at `client/app/api/course/Survey/Responses.js`.